tests/test_utils.py

```diff
@@ -94,3 +98,94 @@ def test_fill_in_from_config(self, mocker):
             cli_utils.fill_in_from_config(
                 "jsonld_none", jsonld_none, mock_keys_invalid
             )
+
+
+class FakeResponse:
+    status: int
+    data: bytes
+
+    def __init__(self, *, data: bytes):
+        self.data = data
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, type, value, traceback):
+        pass
+
+    def read(self):
+        self.status = 200 if self.data is not None else 404
+        return self.data
+
+    def close(self):
+        pass
+
+
+class TestIOUtils:
+
+    def test_json_load(self, tmpdir, mocker):
+
+        json_file = tmpdir.join("example.json")
+        json_file.write_text(json.dumps([
+            {'k1': 'v1'},
+            {'k2': 'v2'}
+        ]), encoding="utf-8")
+
+        with open(json_file, encoding='utf-8') as f:
+            expected = json.load(f)
+
+        result = io_utils.load_json(str(json_file))
+
+        assert result == expected
+
+        mock_urlopen = mocker.patch("urllib.request.urlopen",
+                                    return_value = FakeResponse(data=json.dumps([
+                                        {'k1': 'v1'}, 
+                                        {'k2': 'v2'}]
+                                    ))
+                                    )
+        
+        res = io_utils.load_json("http://example.com")
+        assert res == [
+            {'k1': 'v1'},
+            {'k2': 'v2'}
+        ]
+
+        assert mock_urlopen.call_count == 1
+
+    
+    def test_export_json(self, tmpdir):
+
+        json_str = json.dumps([
+            {'k1': 'v1'},
+            {'k2': 'v2'}
+        ])
+
+        export_file = tmpdir.join("export_json_expected.json")
+        io_utils.export_json(json_str, export_file)
+
+        with open(export_file, encoding='utf-8') as f:
+            expected = json.load(f)
+
+        assert expected == json_str
+
+
+    def test_load_schema(self, tmpdir, mocker):
+
+        jsonld_file = tmpdir.join("example_schema.jsonld")
+        jsonld_str = {
+            '@context': {'k': 'v'},
+            '@graph': [
+                {'k1': 'v1'},
+                {'k2': 'v2'}
+        ]}
+        jsonld_file.write_text(json.dumps(jsonld_str), encoding="utf-8")
+
+        mocker.patch("schematic.LOADER.filename",
+                    return_value=os.path.normpath(jsonld_file))
+
+        expected_default = io_utils.load_default()
+        expected_schemaorg = io_utils.load_schemaorg()
+
+        assert expected_default == jsonld_str
+        assert expected_schemaorg == jsonld_str
```

Comment:
I don't think we need to patch the loader here. Since it's a local file, we should be able to load the actual BioThings and Schema.org JSON-LD models. Obviously, we can't check whether the entire JSON structure is correct, but I think a few spot checks (_e.g._ number of elements in `@graph`, the list of context keys) should suffice for testing the loading.

Reply:
That makes sense @BrunoGrandePhD, thanks. I've patched the `LOADER.filename()` function since we need some way to overwrite the hardcoded schema/file paths that are already set for each of these methods (`load_default()` and `load_schemaorg()`).

The basic checks I've added are:
- list of context keys
- number of graphs (if multiple)
- number of elements in graph